### ISSUE TITLE
Suppress detection for unnecessary 'using' directives if the namespace can't be resolved

### ISSUE THREAD

**Issue by DanielRosenwasser** [dotnet/roslyn#5741]
![image](https://cloud.githubusercontent.com/assets/972891/10326544/63246262-6c51-11e5-9c3c-61de1c801140.png)

If the user is about to add an assembly reference, then it's a little annoying that "Show potential fixes" ends up trying to remove the `using` directive instead of adding the correct assembly.

Instead, just don't try to suggest a fix here and don't report that the `using` directive is unnecessary.

**Comment by gafter** [comment]
This is a "cascaded diagnostic" that should be suppressed in the compiler.

**Comment by CyrusNajmabadi** [comment]
Closing out as "won't fix".  I looked into this, but we have a lot of tests that validate this  behavior, and give reasons for why it is good.  Namely, it's normaly to have something like:

```
using Real.Namespace;
```

Then, a user removes all the acutal types that existed in `Real.Namespace`.  At that point, they will want to get rid of the namespace.  Having these be available for the fixer to remove (including with `fix all in Project/Solution`), is valuable for fixing this up.

While it is understandable that in some cases the user will not want this particular fix, it would still be too harmful to not offer it given all the cases where it is wanted.
